src/python/SConscript

```diff
@@ -37,6 +37,7 @@ PySource('gem5.components.boards', 'gem5/components/boards/__init__.py')
 PySource('gem5.components.boards', 'gem5/components/boards/abstract_board.py')
 PySource('gem5.components.boards', 'gem5/components/boards/mem_mode.py')
 PySource('gem5.components.boards', 'gem5/components/boards/riscv_board.py')
+PySource('gem5.components.boards', 'gem5/components/boards/lupv_board.py')
```

Comment:
For this, I updated the lupio-specs, but for the README, I was under the impression that we'd be putting this information somewhere else since this was supposed to be a temporary repo, but I'm not sure where it should go.  However, I'll go ahead and get rid of the second branch and make sure it's updated.